ibis/backends/snowflake/__init__.py

```diff
@@ -145,7 +145,13 @@ def version(self) -> str:
             return con.exec_driver_sql("SELECT CURRENT_VERSION()").scalar()
 
     def do_connect(
-        self, user: str, password: str, account: str, database: str, **kwargs: Any
+        self,
+        user: str,
+        password: str,
+        account: str,
+        database: str,
+        connect_args: dict[str, Any] = None,
```

Comment:
```suggestion
        connect_args: Mapping[str, Any] | None = None,
```

Reply:
Fixed in 22b57af01f14ba243b9d3d16565fe954fbff24c7. Thanks!